sonar-plugin-api/src/main/java/org/sonar/api/measures/PropertiesBuilder.java

```diff
@@ -19,14 +19,15 @@
  */
 package org.sonar.api.measures;
 
-import org.sonar.api.utils.KeyValueFormat;
-
 import java.util.Map;
 import java.util.TreeMap;
+import org.sonar.api.utils.KeyValueFormat;
 
 /**
  * @since 1.10
+ * @deprecated since 5.6. Use directly {@link KeyValueFormat}.
  */
+@Deprecated
 public class PropertiesBuilder<K, V> {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)